packages/components/src/Modal.purs

```diff
@@ -0,0 +1,29 @@
+module KSF.Modal where
```

Comment:
I guess we aren't using this anywhere?

Reply:
Yes I was already thinking I could remove it from the repo and keep it only locally, should we need to have it later.